src/commands/init.zsh

```diff
@@ -82,19 +82,19 @@ script: zunit"
   # Check that a config file doesn't already exist so that
   # we don't overwrite it
   if [[ -f "$PWD/.zunit.yml" ]]; then
-    echo $(color red "Zunit config file already exists at $PWD/.zunit.yml") >&2
-    exit 1
+    echo $(color yellow "ZUnit config file already exists at $PWD/.zunit.yml. Skipping...")
```

Comment:
is the echo needed? Color should output to stdout by itself from what I can tell